include/cling/Interpreter/Interpreter.h

```diff
@@ -613,6 +613,7 @@ namespace cling {
 
     clang::CompilerInstance* getCI() const;
     clang::Sema& getSema() const;
+    IncrementalParser& getIncrParser() const { return *m_IncrParser; }
```

Comment:
Great, thanks!

Regarding PR #85 I'd really prefer to have a separate dot verb, as this is only the first step of the tracing feature.